spec/spec_helper.rb

```diff
@@ -8,6 +11,11 @@
 ])
 SimpleCov.start
 
+ENV['RACK_ENV'] = 'test'
+
+Capybara.app = Chitter_Main
+
+
```

Comment:
is it also worth putting this in to reset the table to empty? 
RSpec.configure do |config|
  config.before(:each) do
    reset_table